docs/guide/repository.md

```diff
@@ -484,209 +474,76 @@ ezpublish:
-                    - FileSystem
-                inMemory: true
-                registerDoctrineAdapter: false
-                FileSystem:
-                    keyHashFunction: 'crc32'
-    ```
-
-    **This configuration is only recommended for Windows users**.
-
-    Note: You can also define the **path** where you want the cache files to be generated to be able to get even shorter system path for cache files.
-
-##### FileSystem cache backend troubleshooting
-
-By default, Stash Filesystem cache backend stores cache to a sub-folder named after the environment (`i.e. app/cache/dev`, `app/cache/prod`). This can lead to the following issue: if different environments are used for operations, persistence cache (manipulating content, mostly) will be affected and cache can become inconsistent.
-
-To prevent this, there are 2 solutions:
-
-###### 1. Manual
-
-**Always** use the same environment, for web, command line, cronjobs etc.
-
-###### 2. Share stash cache across Symfony environments (prod / dev / ..)
-
-Either by using another Stash cache backend, or by setting Stash to use a shared cache folder that does not depend on the environment.
-
-``` yaml
-# ezplatform.yml
-stash:
-    caches:
-        default:
-            FileSystem:
-                path: "%kernel.root_dir%/cache/common"
-```
-
-This will store stash cache to `app/cache/common.`
-
-##### APC & APCu
-
-This cache backend is using shard memory with APC's user cache feature. As this is per server, it **does not support [multi server (cluster) setups](clustering.md)** .
-
-!!! note "Not supported because of following limitation"
-
-    As APC(u) user cache is not shared between processes, it is not possible to clear the user cache from CLI, even if you set `apc.enable_cli` to On. That is why publishing content from a command line script won't let you properly clear SPI Persistence cache.
-
-Please also note that the default value for `apc.shm_size` is 128MB. However, 256MB is recommended for APC to work properly. For more details please refer to the [APC configuration manual](http://www.php.net/manual/en/apc.configuration.php#ini.apc.shm-size).
-
-**Available settings**
-
-|Setting||
-|-----|-----|
-| `ttl` | The time to live of the cache in seconds; default is 500 (8.3 minutes)                                                                         |
-| `namespace` | A namespace to prefix cache keys with to avoid key conflicts with other eZ Platform sites on same eZ Platform installation; default is `null`. |
-
 #### Redis
 
 This cache backend is using [Redis, a in-memory data structure store](http://redis.io/), via [Redis pecl extension](https://pecl.php.net/package/redis). This is an alternative cache solution for [multi server (cluster) setups](clustering.md), besides using Memcached.
 
-**Available settings**
+See [Redis Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html#configure-the-connection)
```

Comment:
Is it on purpose that this link is to the 3.4 version of the Symfony's doc?
(current = 3.3 and there is a 4.0)

Reply:
yes, 2.0 will be mainly supported with 3.4LTS

Reply:
Ok, thank you for this answer.